### PR TITLE
Update all.json

### PR DESCRIPTION
**Pull request by dubstard**
<html>
<body>

Site | URLScan link
-- | --
ablucryptotokens.com|https://user-images.githubusercontent.com/49607867/166095551-cd2855d5-fb73-4103-a99f-bff7eaae5089.png
accessgains.uk|https://user-images.githubusercontent.com/49607867/166096929-2d8408d5-f69a-4499-9ae5-8e77d1380d86.png
acquiomfinances.com|https://user-images.githubusercontent.com/49607867/166096344-888a4fd7-9c16-4844-911a-932848170755.png
alpha-aerotek.com|https://user-images.githubusercontent.com/49607867/166097060-28f5d632-d7ea-45ca-b7b6-1c470c12e277.png
bittradingenterprises.com|https://user-images.githubusercontent.com/49607867/166096990-fa512bd1-b4d1-4b4c-ae49-3342a4451b45.png
blue-chip.ltd|https://user-images.githubusercontent.com/49607867/166097024-8fa36ebf-82ee-4388-a84f-1ce2885b803e.png
capitaltrusts.ltd|https://user-images.githubusercontent.com/49607867/166097002-056d496e-2c5a-40d2-928b-aabfb4245d25.png
century-trades.com|https://user-images.githubusercontent.com/49607867/166096849-ea311801-20c0-411f-abac-3997a8195f77.png
champagnecapitals.ltd|https://user-images.githubusercontent.com/49607867/166096378-13b2ac69-b3e0-40ec-9945-0a605b36eaa5.png
crypto-booster.biz|https://user-images.githubusercontent.com/49607867/166096537-9de42450-8d8f-4bb2-a998-721f9acbd685.png
crypto-tokenrecovery.com|https://user-images.githubusercontent.com/49607867/166095419-eee4930e-51dc-449c-9780-669d176431a0.png
cryptotrustfx.live|https://user-images.githubusercontent.com/49607867/166096498-46566a5a-3ba8-4c98-8c6c-891a93e426fd.png
expertreliable.co|https://user-images.githubusercontent.com/49607867/166096947-2dfe921c-2452-43e8-abc8-7f42c5b91cb8.png
financial-trading.ltd|https://user-images.githubusercontent.com/49607867/166095431-b4f6387a-23ac-4ff7-a5a1-deccf9b8c41b.png
goldenbridgeassets.com|https://user-images.githubusercontent.com/49607867/166096420-cd659e45-ac72-4303-8eea-e35959dd57be.png
mega-forexchange.ltd|https://user-images.githubusercontent.com/49607867/166095381-0550027a-3aa7-46eb-8762-2713c19cd028.png
otc.eqibank.com|https://user-images.githubusercontent.com/49607867/166096864-601ddb1c-88a6-4787-8b5c-3a74428d93b6.png
safeedgetrade.com|https://user-images.githubusercontent.com/49607867/166096812-87421e12-7815-493c-8c8c-80b0689b5d27.png
standardcryptotraders.com|https://user-images.githubusercontent.com/49607867/166096588-2e3f60f2-2677-4b49-b521-5a3f3ef9b511.png
stepnactivatecode.com|https://user-images.githubusercontent.com/49607867/166095365-c06e7c88-1d76-4e7c-8628-de0f34a3b114.png
upsalefx.com|https://user-images.githubusercontent.com/49607867/166096365-693607e5-f844-4c07-82cf-bac67984e7b1.png
vitexlimited.com|https://user-images.githubusercontent.com/49607867/166096520-00e8093d-e779-4b9d-b654-d0de1670251d.png
walletlinknode.com|https://user-images.githubusercontent.com/49607867/166095348-4200639a-f1bb-4d9a-802d-5f6d697b8c85.png

</body>
</html>

"ablucryptotokens.com",
"coinminers.info",
"connectwalletpancaswap.dynvpn.de",
"crypto-tokenrecovery.com",
"financial-trading.ltd",
"desktopclient.io",
"helpsconnect.live",
"io.metamask.walletverification.in",
"mega-forexchange.ltd",
"metamask.gd",
"metamask.io-verification.com",
"metamask.io.desktopclient.io",
"metamask2022.io",
"metamasklivechat.com",
"metamasks.ca",
"metamaskwebx.com",
"metamaskzendesk.helpsconnect.live",
"stepnactivatecode.com",
"walletlinknode.com",
"walletverification.in",

### CODE REVIEW

**Review by dubstard — COMMENTED** [review]
remove dupes
"metamask2022.io",
"walletverification.in",

**Review by dubstard — COMMENTED** [review]
more
"accessgains.uk",
		"acquiomfinances.com",
		"alpha-aerotek.com",
		"armetatrade.com",
		"autoethcalibrator.com",
		"bittradingenterprises.com",
		"blockube-deluxe.com",
		"blue-chip.ltd",
		"capital-option.co",
		"capitaltrusts.ltd",
		"century-trades.com",
		"champagnecapitals.ltd",
		"crypto-booster.biz",
		"crypto-tokenrecovery.com",
		"cryptotrustfx.live",
		"expertreliable.co",
		"globalpower.ltd",
		"goldenbridgeassets.com",
		"mega-forexchange.ltd",
		"otc.eqibank.com",
		"safeedgetrade.com",
		"securedverifiedoffice.com",
		"standardcryptotraders.com",
		"tesla-exchange.ltd",
		"thegoldasset.org",
		"upsalefx.com",
		"vitexlimited.com",
		"xeroxcapital.com",

[more](https://twitter.com/dubstard/status/1520301285450334211)

**Review by dubstard — COMMENTED** [review]
remove dupes
   "crypto-tokenrecovery.com",
   "mega-forexchange.ltd",
   "autoethcalibrator.com",
   "blockube-deluxe.com",

**Comment by dubstard** [comment]
✅Ready for review
[The Scam sites will soon stop working as well.](https://twitter.com/Namecheap/status/1520303449799340033)

File: all.json [hunk 1]
```diff
@@ -14,6 +14,48 @@
...
 		"1inch.tk",
+		"accessgains.uk",
+		"acquiomfinances.com",
+		"alpha-aerotek.com",
+		"armetatrade.com",
+		"bittradingenterprises.com",
+		"blue-chip.ltd",
+		"capital-option.co",
+		"capitaltrusts.ltd",
+		"century-trades.com",
```
Comment (jacogr) [inline all.json:25]: For all these, i.e. the first batch, I honestly cannot evaluate them - they are all trading platforms, I don't see the "standard" steal-your-seed and have not signed up (checked it though), to form see how they try to take advantage.

These are really out of my wheelhouse.

File: all.json [hunk 2]
```diff
@@ -14,6 +14,48 @@
...
+		"champagnecapitals.ltd",
+		"crypto-booster.biz",
+		"cryptotrustfx.live",
+		"expertreliable.co",
+		"globalpower.ltd",
+		"goldenbridgeassets.com",
+		"otc.eqibank.com",
+		"safeedgetrade.com",
+		"securedverifiedoffice.com",
```
Comment (jacogr) [inline all.json:34]: This one is blocked by Google as well.

File: all.json [hunk 3]
```diff
@@ -14,6 +14,48 @@
...
+		"capitaltrusts.ltd",
+		"century-trades.com",
+		"champagnecapitals.ltd",
+		"crypto-booster.biz",
+		"cryptotrustfx.live",
+		"expertreliable.co",
+		"globalpower.ltd",
+		"goldenbridgeassets.com",
+		"otc.eqibank.com",
+		"safeedgetrade.com",
```
Comment (jacogr) [inline all.json:33]: Same comment as above applies to this group. Have no idea how to evaluate these.

File: all.json (hunk 1 above)
Comment (dubstard) [reply all.json:25]: If you check tiny details about any of them you will find more details regarding a complex vast network of advance fee scams
the way it works is like this
they spam victims via phone calls and lure them to invest
the portfolio grows on theory, but upon any attempt to withdraw the problems begin.

for example the phone number listed on the first one  `armetatrade.com` is exactly the same as 'itierion.com`

Some of them have fake company IDs, or company ID belonging to other entities.
Others go the extra mile to actually register a firm, but always bogus.

Another giveaway is the source code where you can see fake messages like "person from %your country% just invested xyz dollars"

Third giveaway is copy pasta BS - if you copy paste any text passage you will discover hundreds more.

[All the ones listed here are the same scam](https://twitter.com/dubstard/status/1514837690465890310)

File: all.json (hunk 3 above)
Comment (dubstard) [reply all.json:33]: pictures of the team are always stock photos, etc

Comment (dubstard) [reply all.json:33]: image

File: all.json (hunk 1 above)
Comment (dubstard) [reply all.json:25]: another giveaway is 
-social media links are always blank
Investors have same pictures (stock photos - if you Reverse image Search any CEO's picture, you will find 1000 more websites with the same CEO
they are all fake 100%

Comment (dubstard) [reply all.json:25]: image

Comment (dubstard) [reply all.json:25]: image

**Review by dubstard — COMMENTED** [review]
added screenshots and explanation

**Comment by dubstard** [comment]
the worst bit is those are close to impossible to takedown, as registrars do not have a concrete brand to complain for phishing or copyright, so they need proof of monetary loss before they even start an investigation.

**Comment by dubstard** [comment]
the easiest giveaway are the impossible APY / ROI rates.
image

**Comment by jacogr** [comment]
Like I said, this is way out of my wheelhouse to review since I can certainly do the "normal crypto scams" (aka the "normal" metamask-* stuff at the bottom of this PR), but in this case the overall list is just way over my head. So I will have to leave it to others to look at the details since I just don't have any specific insights here.

**Comment by dubstard** [comment]
Thanks for taking the time to review! Shall wait for one of the others from the crew to chip in with their review as well.

**Comment by dubstard** [comment]
This is why all those should be blocked IMO

Concrete example:
Fake Bank (investment scam)
Office: Somewhere in new york london
📞+1234567894
image
image

Literally on the same domain - Wallet stealer (as part of the "investment" fraud scheme)
image

....

**Pull request closed without merging**